style/Theme.ts

```diff
@@ -0,0 +1,27 @@
+const Theme = {
+  color: {
+    primary: "#f1f0f0",
+    secundary: "#005eb4",
+  },
+  font: {
+    small: "0.5rem",
+    default: "1rem",
+    big: "2rem",
+    bigger: "3rem",
+  },
+
+  spacing: {
+    0: "0px",
+    2: "2px",
```

Comment:
one more 💅🏼 thing, when you create variables is always nice to use semantic names
so, if in the future you need to change your spacings to 3/6/9/etc, the variables' names need to change too 

that's why on motriz we use 100/200/300 (and many other DS or style libraries use a scale for variables too), with these generic names, we can change the values without worrying about the semantic

another alternative is to do the same you did with fonts - but finding names can be harder lol, especially if you have a lot of sizes - for that, i would recommend defining a `base`, `medium`, and `wide` value, from there you can create variants with xx-small/x-small/small/large/x-large/xx-large

to be easier to understand, you could have something like:

```
base-xx-small: 4px;
base-x-small: 8px;
base-small: 12px;
base: 16px;
base-large: 24px;
medium-small: 32px;
medium: 40px;
medium-large: 48px;
wide-small: 56px;
wide: 64px;
wide-large: 72px;
wide-x-large: 80px; 
```

_(these names and values are from a project I did in the past lol)_

Reply:
Love the suggestion!